ee/hogai/graph/root/tools/create_and_query_insight.py

```diff
@@ -204,4 +203,4 @@ def is_editing_mode(cls, context_manager: AssistantContextManager) -> bool:
         """
         Determines if the tool is in editing mode.
         """
-        return AssistantTool.EDIT_CURRENT_INSIGHT.value in context_manager.get_contextual_tools()
+        return AssistantTool.CREATE_AND_QUERY_INSIGHT.value in context_manager.get_contextual_tools()
```

Comment:
Why are you removing this? I had split the two tools so we don't need to override them with the same name.